storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/DeleteOperationTest.java

```diff
@@ -93,6 +94,40 @@ public void deleteByQueryAsObservable() {
     }
 
     @Test
+    public void deleteByQueryAsSingle() {
+        TestSubscriber<Changes> changesTestSubscriber = new TestSubscriber<Changes>();
+
+        storIOContentResolver
+                .observeChangesOfUri(TestItem.CONTENT_URI)
+                .take(2)
+                .subscribe(changesTestSubscriber);
+
+        TestItem testItemToInsert = TestItem.create(null, "value");
+        contentResolver.insert(TestItem.CONTENT_URI, testItemToInsert.toContentValues());
+
+        Cursor firstDbState = contentResolver.query(TestItem.CONTENT_URI, null, null, null, null);
+        Assertions.assertThat(firstDbState).hasCount(1);
```

Comment:
In this file, there are two kinds of `assertThat`: from `org.assertj.android.api` and `org.assertj.core.api`. So static import can't be used for both of them.